FracCalc/FC.java

```diff
@@ -0,0 +1,30 @@
+import java.util.*;
+
+public class FC {
+    public static void main(String[] args) {
+        Scanner S = new Scanner(System.in);
+
+        System.out.println("\nPlease input your expression : ");
+        String exp = S.nextLine();
+        
+        /*
+        Match any characters that are provided within the given String, if the string does not contain any of the valid characters continue to else.
+        Using .matches can cut down on any unnecessary methods that would check for valid arithmetic.
+        */
+        if (exp.matches(".*[+-/*].*")){  
+            produceAnswer(exp);
+            System.out.println("\nRESULT : "+ produceAnswer(exp));
+        }
+        else {

```

Comment:
i like this feature a lot! but...won't the expressions almost always have a / since we are usually inputting fractions?